packages/cf-component-card/src/CardSection.js

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme, status }) => ({
+  display: 'table',
+  tableLayout: 'fixed',
+  width: '100%',
+  borderTop: `1px solid ${theme.colorGrayLight}`,
```

Comment:
I'm trying to remember back from migrating the `& + &`. If memory serves me right then it was causing the double border issue and this fixed it?